test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1471,6 +1471,15 @@ def tearDown(self):
         self.log_enter_teardown()
         self.server.cleanup_jobs(runas=ROOT_USER)
         self.stop_proc_monitor()
+
+        for server in self.servers.values():
+            server.cleanup_files()
```

Comment:
shouldn't it be scheduler.cleanup_files() ?

Reply:
Well, I added cleanup_files() in the PBSService base class. I thought in future if there is a need for the server to clean up files, this function can provide that functionality. As of now, the server does nothing when cleanup_files() is called.

Reply:
sorry, i didn't see the scheduler.cleanup_files() line below. That brings another thought: do you need to call scheduler.cleanup_files() for all scheduler objects (in case of multi-sched) ?